server/NucleiClassification/NucleiClassification.py

```diff
@@ -0,0 +1,181 @@
+import os
+import sys
+import json
+import colorsys
+
+import numpy as np
+import pandas as pd
+from sklearn.externals import joblib
+import dask.dataframe as dd
+
+from ctk_cli import CLIArgumentParser
+
+import logging
+logging.basicConfig(level=logging.CRITICAL)
+
+sys.path.append(os.path.normpath(os.path.join(os.path.dirname(__file__), '..')))
+from cli_common import utils as cli_utils # noqa
+
+
+def gen_distinct_rgb_colors(n, seed=None):
+    """
+    Generates N visually distinct RGB colors
+
+    Parameters
+    ----------
+    n : int
+        Number of distinct RGB colors to output
+
+    seed: int or array_like, optional
+        Seed for the random number generator. See documentation of
+        `numpy.random.seed` for more details
+
+    Returns
+    -------
+    colors_list : list
+        A list of n RGB colors
+
+    """
+
+    np.random.seed(seed)
+
+    color_list = []
+
+    h = np.random.random(n)
+    golden_ratio_conjugate = 0.618033988749895
+
+    for i in range(n):
+
+        h += golden_ratio_conjugate
+        h %= 1
+
+        color_list.append(colorsys.hsv_to_rgb(h, 1.0, 1.0))
```

Comment:
This could be replaced with
```python
h = np.random.random()  # (Passing a parameter would return an array)
golden_ratio_conjugate = (np.sqrt(5) - 1) / 2  # (This is an analytic expression for the constant)
# (The hue parameter is automatically taken mod 1)
color_list = [colorsys.hsv_to_rgb(h + i * golden_ratio_conjugate, 1, 1) for i in range(n)]
```